appdev/index.rst

```diff
@@ -3,190 +3,103 @@ App development
+Ubuntu Touch is a fun and vibrant platform for development. Whether your a new developer or an experienced one we have several resources to help you get started.
 
-Click package overview
-^^^^^^^^^^^^^^^^^^^^^^
+Ubuntu Touch supports several different types of apps.
+:ref:`Native apps <nativeapp>` are apps with interfaces made using QML or HTML and their behavior defined in JavaScript, C++, Python, or Go.
+:ref:`Web apps <webapp>` are special containers to run websites in.
 
-Every ``click`` application package must embed at least 3 files:
-
-``manifest.json`` file
-  Contains application declarations such as application name, description, author, framework sdk target, and version.
-
-  Example ``manifest.json`` file::
-
-    {
-        "name": "myapp.author",
-        "title": "App Title",
-        "version": "0.1"
-        "description": "Description of the app",
-        "framework": "ubuntu-sdk-15.04",
-        "maintainer": "xxxx <xxx@xxxx>",
-        "hooks": {
-            "myapp": {
-            "apparmor": "apparmor.json",
-            "desktop": "app.desktop"
-            }
-        },
-
-
-    }
-
-AppArmor profile policy file
-  Contains which policy the app needs to work properly. See `Security and app isolation`_ below for more information on this file.
-
-``.desktop`` file
-  The launcher file will tell UT how to launch the app, which name and icon to display on the home screen, and some other properties.
-
-  Example of ``app.desktop``::
+.. toctree::
+    :maxdepth: 1
+    :name: toc-app-types
 
-    [Desktop Entry]
-    Name=Application title
-    Exec=qmlscene qml/Main.qml
-    Icon=assets/logo.svg
-    Terminal=false
-    Type=Application
-    X-Ubuntu-Touch=true
+    nativeapp/index
+    webapp/index
 
-  Non exhaustive list of properties:
+Tools
+-----
 
-  - Name: Application title has shown in the dash
-  - Exec: Path to the executable file
-  - Icon: Path to the icon to display
-  - Terminal: ``false`` if will not run in terminal window
-  - Type: Specifies the type of the launcher file. The type can be Application, Link or Directory.
-  - X-Ubuntu-Touch: ``true`` to make the app visible
-  - X-Ubuntu-XMir-Enable: ``true`` if your app is built for X
+`Clickable <http://clickable.bhdouglass.com/en/latest/>`_ is a meta-build system for Ubuntu Touch applications that allows you to compile, build, test and publish `click` packages and provides various templates to get you started with app development.
+It is currently the easiest and most convenient way of building click packages for Ubuntu Touch.
+You can use any code editor or IDE that you choose and build your apps from the commandline with Clickable.
 
-.. todo:
-  link to official .desktop specifications
+Alternatively there is the old `Ubuntu SDK IDE <https://docs.ubuntu.com/phone/en/platform/sdk>`__.
+But it is no longer supported by Canonical, and UBports has chosen to not support it either due to lack of manpower.
```

Comment:
Consider rewording this so the sentence doesn't need to start with "But"

Reply:
Even though English is my first language I still suck at it. But I can see about rewriting that 🙂

Reply:
should writers group be engaged in grammar stuff?  And, does this comment right here screw up the project?  We shall see when I press 'add single comment'...

Reply:
Including the writers group on this stuff makes sense to me, I can do technical stuff but not word stuff :+1: